consensus/src/epoch_manager.rs

```diff
@@ -374,10 +374,23 @@ impl EpochManager {
         peer_id: AccountAddress,
         consensus_msg: ConsensusMsg,
     ) -> anyhow::Result<()> {
-        if let Some(event) = self.process_epoch(peer_id, consensus_msg).await? {
+        // we can't verify signatures from a different epoch
+        let unverified_event = self.process_epoch(peer_id, consensus_msg).await?;
```

Comment:
why you create this var? you probably want
```
if let Some(unverified_event) = self.process_epoch
```

Reply:
That was to make the step (and the comment) stand out.
I find it easier to read while matching the specification like this, wdyt?